### DISCUSSION TITLE
oniguruma install field

### DISCUSSION THREAD

**Discussion by kamaitaching** [eclipse-theia/theia#10506] [General]
![image](https://user-images.githubusercontent.com/88232327/144742009-6a4e1359-fbe5-4ae8-822a-98259153a366.png)
oniguruma安装报错

**Comment by msujew** [comment]
Hi @KamaitachiTools ,

what OS are you using to build Theia? It might be that you are missing some dev-dependencies, see the [dev-guide](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites).
